### ISSUE TITLE
RestClient::Exceptions::OpenTimeout: Timed out connecting to server

### ISSUE THREAD

**Issue by bootstraponline** [versioneye/veye#52]
Open timeout is set to 0 due to invalid default logic for setting this value. `nil.to_i` is 0.

```
> $global_options[:open_timeout]
=> nil
> $global_options[:open_timeout].to_i
=> 0
> open_timeout = $global_options[:open_timeout].to_i || 10
=> 0
> > project_api.resource
=> #<RestClient::Resource:0x007fc23c24d118
 @block=nil,
 @options={:timeout=>30, :open_timeout=>0},
```

- [api/resource.rb](https://github.com/versioneye/veye/blob/2bb0fced632cafbaf9e9f5b9b989eb5e14e578b5/lib/veye/api/resource.rb#L12)

**Comment by timgluz** [comment]
Interesting, 
the value of `$global_options[:open_timeout]` should come from the `~/.veye.rc` file or use default values defined at the beginning of the `bin/veye`;


I agree this line should be fixed : ` open_timeout = $global_options[:open_timeout].to_i || 10`

I must inspect closer why GLI gem doesn't use default global values;

**Comment by bootstraponline** [comment]
I'm using veye in a script which is why `bin/veye` is not read.

**Comment by bootstraponline** [comment]
also why does `$global_options` duplicate everything in both a symbol and string version?

**Comment by timgluz** [comment]
solved in v0.3.2!

**Comment by bootstraponline** [comment]
Thanks!
